File: api-project/MovieReservation/MovieReservation/DTO/MovieInfoResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MovieReservation.DTO
{
    public class MovieInfoResponse
    {
        public MovieInfoResponse()
        {
            this.MovieList = new List<Movie>();
        }
        public List<Movie> MovieList { get; set; }
        public int Result { get; set; }
        public string FailureMessage { get; set; }
    }
}

File: angular-proj/src/app/DTO/MovieInfoResponse.ts
import { Movie } from "./Movie";

export class MovieInfoResponse {
    MovieList: Array<Movie>;
    Result: number;
    FailureMessage: string;

    constructor(){
        this.MovieList =  new Array<Movie>();
        this.Result = 0;
        this.FailureMessage = "";
    }
}